cross/tvheadend/Makefile

```diff
@@ -1,7 +1,7 @@
 PKG_NAME = tvheadend
-PKG_VERS = 3.4patch1
+PKG_VERS = 4.0.9
 PKG_EXT = tar.gz
-PKG_DIST_NAME = $(PKG_VERS).$(PKG_EXT)
+PKG_DIST_NAME = v$(PKG_VERS).$(PKG_EXT)
```

Comment:
Is this because `PKG_DIST_NAME` is not `$(PKG_NAME)-$(PKG_VERS)`, e.g., it has the extra `v` in the file name??  Just curious.